build.ps1

```diff
@@ -49,34 +49,3 @@ function bazel() {
     }
     Write-Output "<< bazel $args (ok)"
 }
-
-# ScalaCInvoker, a Bazel worker, created by rules_scala opens some of the bazel execroot's files,
-# which later causes issues on Bazel init (source forest creation) on Windows. A shutdown closes workers,
-# which is a workaround for this problem.
-bazel shutdown
-
-# Prefetch nodejs_dev_env to avoid permission denied errors on external/nodejs_dev_env/nodejs_dev_env/node.exe
-# It isn’t clear where exactly those errors are coming from.
-bazel fetch @nodejs_dev_env//...
-
-bazel build //... `
```

Comment:
I'm all for dropping Windows, but... is this intentional?

Reply:
heh, no, I thought I only checked in the other file, thanks for spotting it.